test/test_distributions.py

```diff
@@ -515,6 +522,8 @@ def test_multinomial_2d(self):
 
     def test_categorical_1d(self):
         p = Variable(torch.Tensor([0.1, 0.2, 0.3]), requires_grad=True)
+        assert is_nan(Categorical(p).mean)
+        assert is_nan(Categorical(p).variance)
```

Comment:
Please use `self.assertTrue` instead of regular assertions. Assert statements can be disabled from command line

Reply:
Nice, I was thinking `assert` looks out of place...